### ISSUE TITLE
[Accessibility] Less than sign is encoded as `lt;` in Chat Accessible View

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#225895]
Type: <b>Bug</b>

CC @meganrogge 

## Reproducible Steps

1. In Copilot Chat View, ask "assign a variable in R?"

1. Open Accessible View

* Notes: `<-` sign is denoted with `lt;-`

VS Code version: Code 1.92.2 (fee1edb8d6d72a0ddff41e5f71a671c23ed924b9, 2024-08-14T17:29:30.058Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.71GB (5.88GB free)|
|Process Argv|. --crash-reporter-id 5c7bafb6-29df-4d8b-b617-c8de21a094b0|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (102)</summary>

Extension|Author (truncated)|Version
---|---|---
android-dev-ext|ade|1.4.0
Bookmarks|ale|13.5.0
openscad|Ant|1.3.1
spellright|ban|3.0.136
zoterolatex|bna|0.4.1
doxdocgen|csc|1.4.0
vscode-eslint|dba|3.0.10
vscode-quick-select|dba|0.2.9
vscode-deno|den|3.38.2
gitlens|eam|15.3.0
EditorConfig|Edi|0.16.4
prettier-vscode|esb|11.0.0
vscode-firefox-debug|fir|2.9.10
shell-format|fox|7.2.5
vscode-google-translate|fun|1.4.13
codespaces|Git|1.17.2
copilot|Git|1.223.1055
copilot-chat|Git|0.19.2024073102
remotehub|Git|0.62.0
vscode-github-actions|git|0.26.3
vscode-pull-request-github|Git|0.94.0
go|gol|0.42.0
overleaf-workshop|iam|0.13.2
cslpreview|igo|0.2.2
latex-workshop|Jam|10.1.0
lilypond-syntax|jea|0.1.1
scheme|jea|0.2.0
better-cpp-syntax|jef|1.27.1
commitlint|jos|2.6.0
google-search|kam|0.0.1
vscode-lua-format|Koi|1.3.8
vscode-js-annotations|lan|0.11.0
lilypond-formatter|lhl|0.2.3
lilypond-pdf-preview|lhl|0.2.8
lilypond-snippets|lhl|0.1.1
vslilypond|lhl|1.7.3
zotero|mbl|0.1.11
git-graph|mhu|1.30.0
vscode-docker|ms-|1.29.2
vscode-edge-devtools|ms-|2.1.5
black-formatter|ms-|2024.2.0
debugpy|ms-|2024.11.2024081401
flake8|ms-|2023.13.12291011
isort|ms-|2023.13.12071014
python|ms-|2024.13.2024081501
vscode-pylance|ms-|2024.8.1
jupyter|ms-|2024.7.0
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.19
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
remote-containers|ms-|0.380.0
remote-ssh|ms-|0.113.1
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.88.2
vscode-remote-extensionpack|ms-|0.25.0
azure-repos|ms-|0.38.0
cmake-tools|ms-|1.18.44
cpptools|ms-|1.21.6
cpptools-extension-pack|ms-|1.3.0
js-debug-nightly|ms-|2024.8.1517
live-server|ms-|0.4.14
powershell|ms-|2024.3.2
remote-explorer|ms-|0.4.3
remote-repositories|ms-|0.40.0
remote-server|ms-|1.5.2
vscode-github-issue-notebooks|ms-|0.0.130
vscode-selfhost-test-provider|ms-|0.3.25
vscode-serial-monitor|ms-|0.12.0
vscode-speech|ms-|0.10.0
vsliveshare|ms-|1.0.5936
autodocstring|njp|0.6.1
pandocciter|not|0.10.4
subtitles-editor|pep|1.1.7
shiny|Pos|1.0.0
shinyuieditor|pos|0.5.0
quarto|qua|1.114.0
r-debugger|RDe|0.5.4
java|red|1.33.0
vscode-xml|red|0.27.1
r|REd|2.8.4
multi-command|ryu|1.6.0
vscode-3dviewer|sle|0.2.2
vscode-deepl|soe|1.1.1
abc-music|sof|0.4.0
lua|sum|3.10.4
latex-utilities|tec|0.4.14
askbard|tpj|1.2.0
cmake|twx|0.0.17
errorlens|use|3.20.0
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.3.1
vscode-conventional-commits|viv|1.25.0
vscode-arduino|vsc|0.6.0
vscode-gradle|vsc|3.16.4
vscode-java-debug|vsc|0.58.0
vscode-java-dependency|vsc|0.24.0
vscode-java-pack|vsc|0.29.0
vscode-java-test|vsc|0.42.0
vscode-maven|vsc|0.44.0
markdown-all-in-one|yzh|3.6.2
grammarly|znc|0.24.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
01bff139:31013167
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
nativerepl1:31104043
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-c:31111717
nativeloc1:31118318
wkspc-ranged-c:31118571

```

</details>

**Comment by meganrogge** [comment]
@mjbvz this is happening with the dompurify sanitize call (see the values of `renderedMarkdown` and `sanitized`. What would you suggest here? 

I'm guessing this is fine in the case of the chat response bc it's actually rendered as markdown so the `&lt` is not shown, but for the accessible view, we render as plain text. 

![Image](https://github.com/user-attachments/assets/07287eb4-6198-4924-b88b-b1a4d782ac40)
